app/renderer/actions/shared.js

```diff
@@ -8,8 +11,23 @@ export function bindClient () {
   /**
    * When we hear back from the main process, resolve the promise
    */
+  ipcRenderer.removeAllListeners('appium-client-command-response');
   ipcRenderer.on('appium-client-command-response', (evt, resp) => {
     // Rename 'id' to 'elementId'
+    let {res, methodName} = resp;
+
+    // Ignore empty objects
+    if (_.isObject(res) && _.isEmpty(res)) {
+      res = null;
+    }
+
+    if (!_.isNull(res) && !_.isUndefined(res)) {
+      notification.success({
+        message: `'${methodName}' ${i18n.t('result')}`,
```

Comment:
It's better to use a template string there. In some languages the order of words might be different